OpenRA.Mods.Common/Activities/Air/Fly.cs

```diff
@@ -105,6 +105,9 @@ public override Activity Tick(Actor self)
 				soundPlayed = true;
 			}
 
+			if (self.IsAtGroundLevel())
```

Comment:
Can you please add a comment here like `// We are taking off, so remove ground influence` to help reduce confusion?

Reply:
Done.